packages/common/mui/src/lib/card-variants/incentive-active/incentive-active.tsx

```diff
@@ -0,0 +1,203 @@
+              <CardInput
+                textAlign="left"
+                marginBottom={theme.spacing(3)}
+                sx={{ color: formattedRate.includes('-') ? colors.red : '' }}
+              >
+                {formattedRate}
+              </CardInput>
+              <SectionTitle textAlign="left" marginBottom={theme.spacing(1)}>
+                <PlusIcon width={'9px'} />
+                <FormattedMessage defaultMessage="NOTE INCENTIVE" />
+              </SectionTitle>
+              <CardInput
+                textAlign="left"
+                marginBottom={theme.spacing(3)}
+                sx={{
+                  color: formattedIncentiveRate.includes('-') ? colors.red : '',
+                }}
+              >
+                {formattedIncentiveRate}
+              </CardInput>
+            </Box>
+            <Box>
+              <SectionTitle textAlign="left" marginBottom={theme.spacing(1)}>
+                <FormattedMessage defaultMessage="Default Terms" />
+              </SectionTitle>
+              <Box sx={{ display: 'flex', alignItems: 'flex-start' }}>
+                <LightningIcon
+                  sx={{ height: '1.5em', marginRight: theme.spacing(1) }}
+                />
+                <H4
+                  textAlign="left"
+                  marginBottom={theme.spacing(4)}
+                  sx={{
+                    color: formattedTotalRate.includes('-') ? colors.red : '',
+                  }}
+                >
+                  {formattedTotalRate}
+                </H4>
+              </Box>
+              <SectionTitle textAlign="left" marginBottom={theme.spacing(1)}>
+                <FormattedMessage defaultMessage="Custom Terms" />
+              </SectionTitle>
+              <Box
+                marginBottom={theme.spacing(4)}
+                sx={{
+                  color: formattedCustomRate.includes('-')
+                    ? colors.red
+                    : theme.palette.typography.main,
+                  display: 'flex',
+                  alignItems: 'baseline',
+                }}
+              >
+                <Label
+                  sx={{
+                    fontWeight: theme.typography.fontWeightRegular,
+                    marginRight: theme.spacing(1),
+                    color: theme.palette.typography.main,
+                  }}
+                >
+                  <FormattedMessage defaultMessage="Up To " />
+                </Label>
+                <LargeInputTextEmphasized>
+                  {formattedCustomRate}
+                </LargeInputTextEmphasized>
+              </Box>
+            </Box>
+          </ContentWrapper>
+        </Box>
+        <Button
+          fullWidth
+          size="large"
+          variant="contained"
+          sx={{
+            textTransform: 'uppercase',
+          }}
+        >
+          {buttonText}
+        </Button>
+      </Card>
+    </Link>
+  );
+};
+
+const ContentWrapper = styled(Box, {
```

Comment:
Can we export these from some file and share them across the different cards? They look to be repeated in multiple files.

Reply:
They're slightly different and only used in the incentive leveraged and fixed cards

Reply:
I imported StyledIcon from the fixed card